openshift-knative-operator/pkg/serving/istio.go

```diff
@@ -7,10 +7,23 @@ import (
 	"knative.dev/operator/pkg/apis/operator/base"
```

Comment:
File will probably needs to be renamed as we cover both net-kourier and net-istio.